client/landing/gutenboarding/onboarding-block/design-selector/index.tsx

```diff
@@ -46,61 +50,74 @@ const DesignSelector: React.FunctionComponent = () => {
 				<div className="design-selector__heading">
 					<Title>{ __( 'Choose a design' ) }</Title>
 					<SubTitle>
-						{ __( 'Pick your favorite homepage layout. You can customize or change it later.' ) }
+						{ isAnchorFmSignup
+							? __(
+									'Pick a homepage layout for your podcast site. You can customize or change it later.'
+							  )
+							: __( 'Pick your favorite homepage layout. You can customize or change it later.' ) }
 					</SubTitle>
 				</div>
 				<ActionButtons>
 					<BackButton onClick={ goBack } />
 				</ActionButtons>
 			</div>
 			<div className="design-selector__design-grid">
-				<div className="design-selector__grid">
-					{ getRandomizedDesigns().featured.map( ( design ) => (
-						<button
-							key={ design.slug }
-							className="design-selector__design-option"
-							data-e2e-button={ design.is_premium ? 'paidOption' : 'freeOption' }
-							onClick={ () => {
-								setSelectedDesign( design );
+				<div
+					className={ isAnchorFmSignup ? 'design-selector__grid-minimal' : 'design-selector__grid' }
+				>
+					{ getRandomizedDesigns()
+						.featured.filter(
```

Comment:
Despite the suggested location for where to add this filter in `client/landing/gutenboarding/available-designs.ts`, podcast sites are not determined via the Config API.

Available designs also get stored on app bootstrap in `client/landing/gutenboarding/index.tsx`, which is also not an appropriate place to add this, since the design step could be in any part of the site creation flow.

Lastly, the filtering is done here so that the available designs remain as the source of truth, and the filtering is done on the view, right before being shown to the user.